README.md

```diff
@@ -0,0 +1,22 @@
+# Make or Break design
+
+In this repository you can find the style and interaction guidelines for Make
+or Break events.
+
+
+## Licensing
+
+Copyright 2019 Make or Break.
+
+The files available here are to be used only as a reference to implement the
+Make or Break website.
```

Comment:
Not sure if necessary, but a link to the project, could be nice.

Reply:
The github project? or the makeorbreak.io website?

Reply:
I was talking about the GitHub project. It would make discoverability for someone who came across this repo.